v2/charts/azure-service-operator/values.yaml

```diff
@@ -155,6 +155,44 @@ networkPolicies:
   # Destination CIDR for talking to PostgreSQL servers
   postgresqlCIDR: 0.0.0.0/0
 
+# Node labels for pod assignment
+# Ref: https://kubernetes.io/docs/user-guide/node-selection/
+nodeSelector:
+  kubernetes.io/os: linux
+
+# podAnnotations contain the pod annotations for Azure Service Operator
+podAnnotations: {}
+
+# Labels to be added to the pod
+podLabels: {}
+
+# The securityContext of the pod
+# See https://kubernetes.io/docs/tasks/configure-pod-container/security-context/#set-the-security-context-for-a-pod
+podSecurityContext: {}
+
+# Assign a PriorityClassName to pods if set
+priorityClassName: ""
+
+# Recomended initial values for resources
+# adjust them as necessary
+resources:
+  limits:
+    cpu: 500m
+    memory: 512Mi
+  requests:
+    cpu: 200m
+    memory: 256Mi
+
+# Number of old history to retain to allow rollback
+# Default Kubernetes value is set to 10
+revisionHistoryLimit: 10
+
+# Specify security settings for a Container
+# Ref: https://kubernetes.io/docs/tasks/configure-pod-container/security-context/#set-the-security-context-for-a-container
```

Comment:
I think there should be a note here that it is _strongly_ recommended you do not increase this pods permissions, and also a call to action to raise an issue if you believe things could be more locked down than they are here.

Then obviously we should lock things down more as we're lacking some defaults like:
```
runAsNonRoot: true
```
(I can do this in a followup PR).

Maybe something like: 
```suggestion
# Ref: https://kubernetes.io/docs/tasks/configure-pod-container/security-context/#set-the-security-context-for-a-container
# It is not recommended to reduce the restrictions in this list, but additional restrictions outside of the default set 
# can be applied. If you believe additional securityContext configuration should be specified by default
# please raise an issue.
```

Reply:
I can add the values from the upper comment if you'd agree with that.